packages/app/components/EventCard/EventCard.js

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback, useEffect, useState} from 'react';
+import { View, Image, Text, FlatList, StyleSheet } from 'react-native';
+import { Card, ListItem, Icon, Divider } from 'react-native-elements';
+import EventCardText from './EventCardText';
+import EventCardRegistration from './EventCardRegistration';
+
+
+
+const styles = StyleSheet.create({
+  banner: {
+    width: 360,
```

Comment:
I recommend looking into relative CSS units like `vw` in this case instead of pixel values. See screenshot of the empty space in the frontend channel.

Reply:
Percentage would be the correct unit, since `vw` would reference the screen size rather than the parent container's width.